integration/map.go

```diff
@@ -32,6 +32,8 @@ import (
 
 const treeID = int64(0)
 
+var hasher = maphasher.Default
```

Comment:
I moved the hasher variable to a package variable in order to make testing with a different hasher easier - one place to change it.